collect_app/src/main/java/org/odk/collect/android/activities/FormEntryActivity.java

```diff
@@ -563,7 +576,7 @@ private void nonblockingCreateSavePointData() {
     }
 
     @Nullable
-    private FormController getFormController() {
+    public FormController getFormController() {
```

Comment:
I think it's better to leave this method private and just use `Collect.getInstance().getFormController()` wherever you need.